python/cugraph/core/core_number.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cugraph.core import core_number_wrapper
+
+
+def core_number(G):
+    """
+    Compute the core numbers for the nodes of the graph G.
```

Comment:
Maybe a bit more description here.  NetworkX has a couple of sentences describing k-core and what the core number really is.  I couldn't remember and had to go look it up.